services/management/dbaas/psmdb_cluster_service.go

```diff
@@ -0,0 +1,193 @@
+	if err != nil {
+		return nil, err
+	}
+
+	in := dbaascontrollerv1beta1.ListPSMDBClustersRequest{
+		KubeAuth: &dbaascontrollerv1beta1.KubeAuth{
+			Kubeconfig: kubernetesCluster.KubeConfig,
+		},
+	}
+
+	out, err := s.controllerClient.ListPSMDBClusters(ctx, &in)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Can't get list of PSMDB clusters: %s", err.Error())
+	}
+
+	clusters := make([]*dbaasv1beta1.ListPSMDBClustersResponse_Cluster, len(out.Clusters))
+	for i, c := range out.Clusters {
+		cluster := dbaasv1beta1.ListPSMDBClustersResponse_Cluster{
+			Name: c.Name,
+			Params: &dbaasv1beta1.PSMDBClusterParams{
+				ClusterSize: c.Params.ClusterSize,
+				Replicaset: &dbaasv1beta1.PSMDBClusterParams_ReplicaSet{
+					ComputeResources: &dbaasv1beta1.ComputeResources{
+						CpuM:        c.Params.Replicaset.ComputeResources.CpuM,
+						MemoryBytes: c.Params.Replicaset.ComputeResources.MemoryBytes,
+					},
+				},
+			},
+		}
+
+		clusters[i] = &cluster
+	}
+
+	return &dbaasv1beta1.ListPSMDBClustersResponse{Clusters: clusters}, nil
+}
+
+// ShowPSMDBCluster returns a PSMDB cluster by name.
+func (s PSMDBClusterService) ShowPSMDBCluster(ctx context.Context, req *dbaasv1beta1.ShowPSMDBClusterRequest) (*dbaasv1beta1.ShowPSMDBClusterResponse, error) {
+	kubernetesCluster, err := models.FindKubernetesClusterByName(s.db.Querier, req.KubernetesClusterName)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: implement on dbaas-controller side:
+	// 1. Get psmdb host and status
+	//  - Ex.: kubectl get -o=json PerconaServerMongoDB/<cluster_name>
+	// 2. Get root password:
+	//   - Ex.: kubectl get secret my-cluster-name-secrets -o json  | jq -r ".data.MONGODB_USER_ADMIN_PASSWORD" | base64 -d
+	_ = kubernetesCluster
+	resp := dbaasv1beta1.ShowPSMDBClusterResponse{
+		Name:     req.Name,
+		Username: "userAdmin",
+		Password: "userAdmin123456",
+		Host:     fmt.Sprintf("%s-rs0.default.svc.cluster.local", req.Name),
+		Port:     3306,
+	}
+
+	return &resp, nil
+}
+
+// CreatePSMDBCluster creates PSMDB cluster with given parameters.
+//nolint:dupl
+func (s PSMDBClusterService) CreatePSMDBCluster(ctx context.Context, req *dbaasv1beta1.CreatePSMDBClusterRequest) (*dbaasv1beta1.CreatePSMDBClusterResponse, error) {
+	kubernetesCluster, err := models.FindKubernetesClusterByName(s.db.Querier, req.KubernetesClusterName)
+	if err != nil {
+		return nil, err
+	}
+
+	in := dbaascontrollerv1beta1.CreatePSMDBClusterRequest{
+		KubeAuth: &dbaascontrollerv1beta1.KubeAuth{
+			Kubeconfig: kubernetesCluster.KubeConfig,
+		},
+		Name: req.Name,
+		Params: &dbaascontrollerv1beta1.PSMDBClusterParams{
+			ClusterSize: req.Params.ClusterSize,
+			Replicaset: &dbaascontrollerv1beta1.PSMDBClusterParams_ReplicaSet{
+				ComputeResources: &dbaascontrollerv1beta1.ComputeResources{
+					CpuM:        req.Params.Replicaset.ComputeResources.CpuM,
+					MemoryBytes: req.Params.Replicaset.ComputeResources.MemoryBytes,
+				},
```

Comment:
Let's make `ComputeResources` optional - this reflect that it can be optional in yaml, and simplify testing and make it really optional on UI?